Week-6/Exercise6_2.js

```diff
@@ -0,0 +1,43 @@
+function printSpiral(a, m, n) {
+    let k = 0,
+        l = 0,
+        i;
+
+    while (k < m && l < n) {
```

Comment:
instead of m and n, names of these variable can be enhanced with proper names like cols, rows. while doing competitive coding, these variables will make sense as you don't need to explain your program to anyone, while raising a PR, naming of variable matters.